src/layout/actions/layout.rs

```diff
@@ -224,6 +254,36 @@ impl LayoutTree {
         self.validate();
     }
 
+    /// Gets the active container and toggles it based on the following rules:
+    /// * If horizontal, make it vertical
+    /// * else, make it horizontal
+    /// This method does *NOT* update the actual views geometry, that needs to be
+    /// done separately by the caller
+    pub fn toggle_active_horizontal(&mut self) {
```

Comment:
`switch_active_container_layout` is confusing, because it doesn't switch it, it'll only toggle between horizontal/vertical. It would be confusing within the context of later layout types